### PR TITLE
alphabetically order menus/sub menus

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-Do:

- [x] Order Menus
- [x] Order Sub Menus

### CODE REVIEW

**Comment by SidelineCory24** [comment]
image

**Comment by elizabeth-jimenez** [comment]
@SidelineCory24 Did the ticket ask for Dashboards to remain at the top?

**Comment by SidelineCory24** [comment]
> @SidelineCory24 Did the ticket ask for Dashboards to remain at the top?



> @SidelineCory24 Did the ticket ask for Dashboards to remain at the top?

Yes it did. I missed that in the ordering and it has been updated.
image

**Comment by elizabeth-jimenez** [comment]
I would've just modified our Menu.js - this way is MUCH better! Future proof

**Comment by elizabeth-jimenez** [comment]
I would've just modified our Menu.js - this way is MUCH better! Future proof

File: src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx [hunk 1]
```diff
@@ -53,6 +53,15 @@ const ProfileMenuExpanded = (props) => {
...
   };
 
+  const getProfileMenuSort = filter(GET_PROFILE_MENU(), { text: 'Profile' });
+  const getProfileMenuSort$ = sortBy(remove(GET_PROFILE_MENU(),
+    menu => menu.text !== 'Profile'), [(menu) => menu.text.toLowerCase()],
+  );
+
+  getProfileMenuSort$.forEach((item) => {
+    getProfileMenuSort.push(item);
+  });
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx:63]: could do this instead:



  const getProfileMenuSort = sortBy(remove(GET_PROFILE_MENU(),
    menu => menu.text !== 'Profile'), [(menu) => menu.text.toLowerCase()],
  );

  getProfileMenuSort.unshift(filter(GET_PROFILE_MENU(), { text: 'Profile' }));

Comment (elizabeth-jimenez) [inline src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx:63]: could do this instead:



  const getProfileMenuSort = sortBy(remove(GET_PROFILE_MENU(),
    menu => menu.text !== 'Profile'), [(menu) => menu.text.toLowerCase()],
  );

  getProfileMenuSort.unshift(filter(GET_PROFILE_MENU(), { text: 'Profile' }));

Comment (elizabeth-jimenez) [reply src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx:63]: wait, looking into this, Profile is gone lol

Comment (elizabeth-jimenez) [reply src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx:63]: wait, looking into this, Profile is gone lol

Comment (SidelineCory24) [reply src/Components/ProfileMenu/ProfileMenuExpanded/ProfileMenuExpanded.jsx]: done

**Comment by elizabeth-jimenez** [comment]
Cory, :( im sorry, I didn't realize until you added ` || {})` to the unshift that the edge case of not having `Dashboard` as a submenu was possible - in that case, i think the filter you had before works better rip - will discuss with u

**Comment by SidelineCory24** [comment]
> Cory, :( im sorry, I didn't realize until you added ` || {})` to the unshift that the edge case of not having `Dashboard` as a submenu was possible - in that case, i think the filter you had before works better rip - will discuss with u

discussed offline and updates were made/pushed

**Pull request merged**